inception-pdf-editor/src/test/java/de/tudarmstadt/ukpinception/pdfeditor/pdfanno/model/PdfAnnoRendererTest.java

```diff
@@ -193,17 +196,19 @@ public void tetsConvertToDocumentOffset() throws Exception
         state.setFirstVisibleUnit(WebAnnoCasUtil.getFirstSentence(jCas));
         state.setProject(project);
 
-        Offset docOffsetKarin = PdfAnnoRenderer
-            .convertToDocumentOffset(cas.getDocumentText(), pdfExtractFile, new Offset(3, 7));
-        assertThat(docOffsetKarin).isEqualTo(new Offset(0, 5));
-        Offset docOffsetFliegt = PdfAnnoRenderer
-            .convertToDocumentOffset(cas.getDocumentText(), pdfExtractFile, new Offset(8, 13));
-        assertThat(docOffsetFliegt).isEqualTo(new Offset(6, 12));
-        Offset docOffsetSie = PdfAnnoRenderer
-            .convertToDocumentOffset(cas.getDocumentText(), pdfExtractFile, new Offset(28, 30));
-        assertThat(docOffsetSie).isEqualTo(new Offset(29, 32));
-        Offset docOffsetDort = PdfAnnoRenderer
-            .convertToDocumentOffset(cas.getDocumentText(), pdfExtractFile, new Offset(35, 38));
-        assertThat(docOffsetDort).isEqualTo(new Offset(38, 42));
+        DocumentModel documentModel = new DocumentModel(cas.getDocumentText());
```

Comment:
Can you add line comments and/or a JavaDoc comment explaining what you are testing here and how it is being tested?

Reply:
is it ok like this?